core/domain/exp_services.py

```diff
@@ -2834,3 +2834,69 @@ def rollback_exploration_to_safe_state(exp_id):
         caching_services.delete_multi(
             caching_services.CACHE_NAMESPACE_EXPLORATION, None, [exp_id])
     return last_known_safe_version
+
+
+def can_exploration_be_curated(exploration):
+    """Checks if the exploration be used as the chapter of a story.
```

Comment:
exploration can

Reply:
I think this comment has become invalid now.